easybuild/easyconfigs/j/Java/Java-1.8-aarch64.eb

```diff
@@ -0,0 +1,15 @@
+name = 'Java'
+version = '1.8'
+versionsuffix = '-aarch64'
+
+homepage = 'https://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u265-b01/']
```

Comment:
The `8u265-b01` part should be reflected via `versionsuffix`?

Reply:
So you want the `versionsuffix` to be `-aarch64-8u265-b01` (or `-8u265-b01-aarch64`)?

Reply:
I would say the version string should minimum include
```
version = '1.8_265'
```
and the remaining, aarch64, OpenJDK, and build(?) number should be part of the either the version or versionsuffix. So maybe
```
version = '1.8_265-b01-OpenJDK'
versionsuffix = '-aarch64'
```
or
```
version = '1.8_265'
versionsuffix = '-b01-OpenJDK-aarch64'
```